man/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ endmacro(MARIADB_SYMLINK_MANPAGE)
 macro(MARIADB_SYMLINK_MANPAGE_CREATE mysqlname index mannr dir comp)
   LIST(GET MARIADB_SYMLINK_TOS ${index} _mariadbname)
   SET(dest "${CMAKE_CURRENT_BINARY_DIR}/${_mariadbname}.${mannr}")
-  FILE(WRITE ${dest} ".so ${mysqlname}.${mannr}")
+  FILE(WRITE ${dest} ".so man${mannr}/${mysqlname}.${mannr}")
```

Comment:
Why is this important? To be more specific?
Could instead of `man${mannr}` be `${dir}` ?

Reply:
What does ${dir} contain? I can use that variable if it has the correct value.

Reply:
Looking the call of `MARIADB_SYMLINK_MANPAGE_CREATE` macro above: 
```
MARIADB_SYMLINK_MANPAGE_CREATE(${_manname} ${_index} 1 "man1" ManPagesTest)
```
 it can be `man1` or `man8` what we need for symlinks right ?

Reply:
Current `man${mannr}`ensures it is correctly `man1` or `man8` depending on the man page itself. But I can change it to use the dir parameter from the macro.